documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -1013,6 +1014,14 @@ Currently Infinispan provides the following implementations of EntryMergePolicy:
 
 | MergePolicies.REMOVE_ALL
 | Always remove a key from the cache when a conflict is detected.
+
+| MergePolicies.NONE
+| No attempt is made to resolve conflicts. Instead, entries hosted on the minority partition are removed and the nodes in this partition do not hold any data until the rebalance starts. Note, this behaviour is equivalent to prior Infinispan versions which did not support conflict resolution.
+Note, in this case all changes made to one (the part with less nodes) side of the split are lost. But the state after rebalancing will be consistent.
```

Comment:
I think we can combine the last two sentences like so: "In this case all changes made to entries hosted on the minority partition are lost, but once the rebalance has finished all entries will be consistent"

We have already referred to the minority partition, so we should do so again.